content/en/docs/instrumentation/go/manual.md

```diff
@@ -297,7 +297,389 @@ serializing the context.
+
+- Metrics API: [`go.opentelemetry.io/otel/metric`][]
+- Metrics SDK: [`go.opentelemetry.io/otel/sdk/metric`][]
+
+### Initialize Metrics
+
+{{% alert color="info" %}} If you’re instrumenting a library, skip this step.
+{{% /alert %}}
+
+To enable [metrics](/docs/concepts/signals/metrics/) in your app, you'll need to
+have an initialized
+[`MeterProvider`](/docs/concepts/signals/metrics/#meter-provider) that will let
+you create a [`Meter`](/docs/concepts/signals/metrics/#meter).
+
+If a `MeterProvider` is not created, the OpenTelemetry APIs for metrics will use
+a no-op implementation and fail to generate data. Therefore, you have modify the
+source code to include the SDK initialization code using [`go.opentelemetry.io/otel`][],
+[`go.opentelemetry.io/otel/sdk/metric`][], [`go.opentelemetry.io/otel/sdk/resource`][],
+[`go.opentelemetry.io/otel/exporters/stdout/stdoutmetric`][] packages.
+
+Ensure you have the right Go modules installed:
+
+```sh
+go get go.opentelemetry.io/otel \
+  go.opentelemetry.io/otel/exporters/stdout/stdoutmetric \
+  go.opentelemetry.io/otel/sdk \
+  go.opentelemetry.io/otel/sdk/metric
+```
+
+Then initialize a resources, metrics exporter and provider:
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutmetric"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+)
+
+func main() {
+	res, err := newResource()
+	if err != nil {
+		panic(err)
+	}
+
+	meterProvider, err := newMeterProvider(res)
+	if err != nil {
+		panic(err)
+	}
+
+	// Set the global meter provider.
+	otel.SetMeterProvider(meterProvider)
+
+	// Handle shutdown properly so nothing leaks.
+	defer func() {
+		if err := meterProvider.Shutdown(context.Background()); err != nil {
+			log.Println(err)
+		}
+	}()
+}
+
+func newResource() (*resource.Resource, error) {
+	return resource.Merge(resource.Default(),
+		resource.NewWithAttributes(semconv.SchemaURL,
+			semconv.ServiceName("dice"),
+			semconv.ServiceVersion("0.1.0"),
+		))
+}
+
+func newMeterProvider(res *resource.Resource) (*metric.MeterProvider, error) {
+	metricExporter, err := stdoutmetric.New()
+	if err != nil {
+		return nil, err
+	}
+
+	meterProvider := metric.NewMeterProvider(
+		metric.WithResource(res),
+		metric.WithReader(metric.NewPeriodicReader(metricExporter,
+			// Default is 1m. Set to 3s for demonstrative purposes.
+			metric.WithInterval(3*time.Second))),
+	)
+	return meterProvider, nil
+}
+```
+
+Now that a `MeterProvider` is configured, you can acquire a `Meter`.
+
+### Acquiring a Meter
+
+Anywhere in your application where you have manually instrumented code you can
+call [`otel.Meter`](https://pkg.go.dev/go.opentelemetry.io/otel#Meter) to acquire
+a meter. For example:
+
+```go
+import "go.opentelemetry.io/otel"
+
+var meter = otel.Meter("my-service-meter")
```

Comment:
In the tracing manual instrumentation we don't do this (using just the API to acquire the tracer). I wrote those docs ages ago, though, and they were approved then but it's maybe worth re-addressing in a separate PR. What do you think? I'll submit that PR separately.

Reply:
In "Getting a Tracer" section we probably suggest doing to much and it should be splitted into something like "Initialize Tracing" and "Aquire a Tracer". For sure it should be in a seperate PR. I already seen some things in "tracing" sections that should be probably changed/improved but I want this PR to only focus on translating the "metrics" from JS to Go. Then I also want to improve the "Getting Started" for Go. After these two PRs are merged, I think it would be easier to consolidate Tracing and Metrics docs. Maybe we could even change the code snippets so that they refer somehow to the "Roll Dice" service?